infrastructure/ansible/roles/traffic_portal/defaults/main.yml

```diff
@@ -196,7 +196,7 @@ tp_default_properties_template:
         customLink:
           _comment: Do you want to show a custom link to your own ds dashboards?
           show: false
-          baseUrl: "{{ tp_ts_base_url }}/dashboard/script/traffic_ops_deliveryservice.js?which="
+          baseUrl: "{{ tp_ts_base_url }}/dashboard/db/cdn-overview?var-svc="
```

Comment:
`traffic_ops_deliveryservice.js` is in our repo at [/traffic_stats/grafana/traffic_ops_deliveryservice.js](https://github.com/apache/trafficcontrol/blob/048a1ea016/traffic_stats/grafana/traffic_ops_deliveryservice.js). If the `cdn-overview` dashboard is not publicly available, I would be hesitant to reference it here.